src/screencast/screencast.ts

```diff
@@ -388,6 +399,16 @@ export class Screencast {
         });
     }
 
+    private getClipboardContents(): void {
+        encodeMessageForChannel(msg => vscode.postMessage(msg, '*'), 'readClipboard');
+    }
+
+    private pasteClipboardContents(message: string) {
+        this.cdpConnection.sendMessageToBackend('Runtime.evaluate', {
+            expression: `document.execCommand("insertText", false, "${message}");`,
```

Comment:
We should escape any double-quote `"` characters in `message` to avoid a script error in the `Runtime.evaluate` call.

Reply:
Good catch.  Fixed in latest patch.